package.json

```diff
@@ -20,7 +20,7 @@
     "build:index": "babel-node devtools/buildIndexFiles.js",
     "config:lint": "eslint 'config/**' --ext .js,.jsx --env browser,node",
     "config:lint:fix": "eslint --fix 'config/**' --ext .js,.jsx --env browser,node",
-    "deploy": "gh-pages -d build",
+    "deploy": "cp build/index.html build/404.html && gh-pages -d build",
```

Comment:
Could we add the `cp` statement to the end of the `build-doc` script instead?  We reduced the `deploy` script to just the `gh-pages` command in order to pass additional arguments to it (from `publish.sh`) and I don't believe that will work if this is two separate commands.